test/unit_tests/aievec_tests/i16xi16_gemm/gemm64_int16_unroll32.mlir

```diff
@@ -1,5 +1,6 @@
 // REQUIRES: valid_xchess_license
 // RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=32" -aieml=true --aie-vectorize | aie-translate -aieml=true --aievec-to-cpp -o gen_aie-ml.cc
+// RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=32" -redundant-load-store-optimization --convert-vector-to-aievec="aie-target=aieml" -lower-affine | aie-translate -aieml=true --aievec-to-cpp -o gen_aie-ml.cc
```

Comment:
Notice that "-redundant-load-store-optimization" doesn't exist any more, since it's a private pass. Trying to invoke the pass from the command line should fail. For some reason, though, if you run "check-aie" it doesn't, which makes me think this test is not being run, for some reason. Could you look into it?

Reply:
Sorry, I forgot to add the changes for these two unit test cases.